### DISCUSSION TITLE
[@xstate/inspect]: Allow for passing 'events' to `inspect()`

### DISCUSSION THREAD

**Discussion by mattpocock** [statelyai/xstate#2068] [Ideas]
Context: https://github.com/mattpocock/xstate-catalogue/issues/6#issuecomment-814185008

As discussed in xstate-catalogue, it would be great to be able to provide default events to the `inspect()` call we get from `@xstate/inspect`.  For instance:

```
inspect({
  events: {
    CHANGE: {
      value: 'string',
    }
  }
})
```

This means that, when someone clicks 'CHANGE' in the visualiser, it would use the default event type passed into `events`.

**Comment by Andarist** [comment]
`inspect` is a global function and this sort of thing would be better put closer to particular machines. I believe @davidkpiano wants to add a lot of stuff like this - so example payloads could be represented together with a machine.

**Reply by davidkpiano** [reply]
Yeah, the idea is to have something like this:

```js
createMachine({
  // ...
  schema: {
    events: {
      CHANGE: { properties: { value: { type: 'string' } } }
    }
  }, 
  // ...
});
```

Which, in the future, would cause the interpreter to pop up a mini form that allows those parameters to be filled in.
